httpclient5/src/main/java/org/apache/hc/client5/http/cookie/Cookie.java

```diff
@@ -126,5 +127,15 @@
      */
     Date getCreationDate();
 
+    /**
+     * Checks whether this Cookie has been marked as <i>HttpOnly</i>.
+     *
+     * @return true if this Cookie has been marked as <i>HttpOnly</i>,
+     * false otherwise
+     *
+     * @since 5.1
+     */
+    boolean isHttpOnly();
```

Comment:
@arturobernalg We cannot add new methods to interfaces until Java 1.8 upgrade. This change-set will have to wait until 5.2

Reply:
Hi @ok2c 
Thats ok. Since tag version update to 5.2
TY